Tests/ContextSyncTests.swift

```diff
@@ -118,32 +118,32 @@ final class ContextSyncTests: TestCase {
             companies = self.generateDataInContext(childContext, companiesCount: 3, employeesCount: 3)
         }
         let companyNames = companies.map { $0.name }
-
+        
         // WHEN: we save the child context
-        expectation(forNotification: Notification.Name.NSManagedObjectContextDidSave.rawValue, object: childContext, handler: nil)
+        expectation(forNotification: Notification.Name.NSManagedObjectContextDidSave, object: childContext, handler: nil)
         saveContext(childContext) { (result) -> Void in
             XCTAssertTrue(result == .success)
         }
-
+        
```

Comment:
Yeah, but seriously, isn't Xcode supposed to do this for you, too? I've never seen this setting actually do anything:

![settings](https://user-images.githubusercontent.com/4802/28758034-757f9ed0-75d3-11e7-982c-abbc7c91e8ac.png)